extensions/wikia/AdEngine/js/template/bfaa.js

```diff
@@ -134,9 +135,8 @@ define('ext.wikia.adEngine.template.bfaa', [
 		uapContext.setUapId(params.uap);
 		unblockedSlots.forEach(btfBlocker.unblock);
 
-		if (skin === 'mercury') {
-			win.dispatchEvent(new Event('wikia.uap'));
-		}
+		event.initCustomEvent('wikia.uap', true, true, {});
```

Comment:
```
var event = doc.createEvent('CustomEvent');
event.initCustomEvent('wikia.uap', true, true, {});
win.dispatchEvent(event);
```
are lines which showed up twice in this PR. Do you think it makes sense extracting them and use one and the same function in those two places:question: